ktor-client/ktor-client-curl/desktop/interop/include/curl/curlver.h

```diff
@@ -32,14 +32,13 @@
 
 /* This is the version number of the libcurl package from which this header
    file origins: */
-#define LIBCURL_VERSION "8.11.1"
+#define LIBCURL_VERSION "8.17.0-DEV"
```

Comment:
Is it fine that it's the `DEV` version?

Reply:
It is because vcpkg checks out curl sources by tag ([for example curl-8_17_0](https://github.com/curl/curl/blob/curl-8_17_0/include/curl/curlver.h#L35)). The stable version and timestamp are set in [maketgz](https://github.com/curl/curl/blob/curl-8_17_0/scripts/maketgz) script. Which doesn't seem to be called by vcpkg
So the version is actually stable, but `LIBCURL_VERSION` and `LIBCURL_TIMESTAMP` are not correct.